module/settings.mjs

```diff
@@ -339,6 +339,22 @@ export function registerSystemSettings() {
     type: Boolean,
     default: true
   });
+
+  if (game.release.generation >= 12) {
```

Comment:
If this does go in it probably won't be until `4.0` which is dropping V11 support, so no need to worry about backwards compatibility.

Reply:
I had hopes it was relatively uncontroversial.

Reply:
Well, you can leave these in for now and we'll see what Kim thinks. If it doesn't go in `3.3.1` then I imagine it will have to wait for `4.0` cause I think we want to move onto working on that very quickly due to the hard deadline.